code/sound/audiostr.cpp

```diff
@@ -1026,20 +1026,6 @@ void audiostream_set_volume_all(float volume, int type)
 	}
 }
 
-void audiostream_set_volume(int i, float volume)
```

Comment:
This might still be useful so it should be kept.

Reply:
ok